src/ape/types/__init__.py

```diff
@@ -280,6 +280,10 @@ def validate_address(cls, value):
     def block(self) -> "BlockAPI":
         return self.chain_manager.blocks[self.block_number]
 
+    @property
+    def timestamp(self) -> int:
```

Comment:
keep in mind, if anyone was relying on the `__getattr__` approach to checking for a custom event property `timestamp`, this will break their stuff. We have work arounds in place for those scenarios of course, but I am always weary about adding properties to our classes that override `__getattr__`.

Reply:
As noted in my PR note, I don't think this is possible because `timestamp` is a reserved keyword

Reply:
Originally, I was hoping to allow `<field>.<member>` be a supported column name (which would do `__getattr_(field)` and then `__getattr__(member)` on that)

Reply:
Originally? lol

Reply:
More generally I guess haha, I don't think this works (or works well) today

Reply:
I am still good with this PR though.
Though it is a good data science problem: How often is `timestamp` used? What are the chances of breaking existing ape projects? Probably pretty low, and in that case they'd find the PR, hopefully see this comment, and see that you can do this as a work-around

```python
actual_timestamp = log.event_arguments["timestamp"]
assert something with actual_timestamp
```

Reply:
codeslaw.app might help us here

Reply:
I don't think there are many examples of this